book/website/communication/peer-review/peer-review-conflicts.md

```diff
@@ -0,0 +1,83 @@
+(cm-pr-conflicts)=
+# Conflict of Interest
+
```

Comment:
This page might benefit from a short introduction about what the article introduces, something like: "This page will discuss examples of conflicts of interest in the context of research and academic publishing. "